Magento_Catalog/templates/product/widget/viewed/list.phtml

```diff
@@ -76,28 +76,28 @@
         ?>
         <?= $sliderBlockBefore->toHtml(); ?>
 
-        <?= $block->renderApp([
-            'widget_columns' => [
-                'displayMode' => 'grid',
-                'displayAs' => 'slider',
-                'itemWrapperClasses' => 'slider__item',
-                'itemClasses' => 'product-grid-item product-grid-item--slider-item'
-            ],
-            'image' => [
-                'imageCode' => 'recently_viewed_products_grid_content_widget'
-            ]
-        ]);
-        ?>
-
-        <?php
-        $sliderBlockAfter = $this->getLayout()
-            ->createBlock("Magento\Framework\View\Element\Template")
-            ->setTemplate("Magento_Theme::html/slider.phtml");
+        <?= $block->escapeHtml(
+            $block->renderApp([
+                'widget_columns' => [
+                    'displayMode' => 'grid',
+                    'displayAs' => 'slider',
+                    'itemWrapperClasses' => 'slider__item',
+                    'itemClasses' => 'product-grid-item product-grid-item--slider-item'
+                ],
+                'image' => [
+                    'imageCode' => 'recently_viewed_products_grid_content_widget'
+                ]
+            ])
+        )?>
 
-        $sliderBlockAfter->setData([
-            'slider_html'   => 'after-slides'
-        ]);
-        ?>
+        <?php $block->escapeHtml(
```

Comment:
First this way of using escapeHtml is not correct. 

This code here is passing wrong arguments. It passes "variable assignment" and method from first argument (new variables). I guess now you see what's wrong here. If not ask me on chat so I will try to explain better.


Second we should remove this escapeHtml and add noEscape instead of escapeHtml to line ` <?= $sliderBlockAfter->toHtml(); ?>`